### ISSUE TITLE
Obtaining the toolchain as per the README

### ISSUE THREAD

**Issue by tmcarr** [canselcik/libremarkable#59]
I tried to use the link in the readme to obtain the toolchain from remarkable, but the link just redirects to the front page. Is there a new URL to get this?

**Comment by Eeems** [comment]
Take a look at the information here: https://remarkablewiki.com/devel/toolchain

**Comment by tmcarr** [comment]
Yeah, I was pointed there on the discord. I just figured it might also be useful to update the readme. I can do that if you want.

**Comment by Eeems** [comment]
I'm sure @canselcik wouldn't mind a PR :)

**Comment by tmcarr** [comment]
Done in #60 :)
